tox.ini

```
@@ -77,6 +77,7 @@ deps =
     oldestdeps: numpy==1.23.*
     oldestdeps: matplotlib==3.3.*
     oldestdeps: asdf-astropy==0.3.*
+    oldestdeps: attrs < 24.1.0 # can be removed when asdf<3.4 is dropped
```

Easier to grep this way
```suggestion
    oldestdeps: attrs < 24.1.0  # ASDF_LT_3_4
```